src/parserindexer/indexer.py

```diff
@@ -43,20 +43,30 @@ def parse_lpsc_from_path(path_str):
         return doc_id, year, doc_url
     return None, None
 
-def map_basic(doc, noalter_prefix="ner"):
+
+# KW: indexer.py isn't handling the 'sentences' field correctly;
+# Solr ingestion fails.  I'm not sure we need it.  
+# So for now, omitting it.
+def map_basic(doc, noalter_prefix=["ner","rel"], nomap_prefix=["sentences"]):
```

Comment:
Thanks, It makes sense now. I previously thought you tried to index sentences and solr had rejected it.